charts/osm/templates/osm-injector-hpa.yaml

```diff
@@ -10,6 +10,7 @@ spec:
     kind: Deployment
     name: osm-injector
   minReplicas: {{.Values.OpenServiceMesh.injector.autoScale.minReplicas}}
+  # no max limit set for maxReplicas
```

Comment:
I think this comment and the other one like this are a bit confusing. I don't think we need them, so I'd leave them out.